DANS-KNAW/easy-deposit-api
# suggestions for improved logging

Fixes EASY-

- [ ] finalize draft

#### When applied it will
* log requests when received and responses when sent
  not (only) when the module starts/completes processing
* 

#### Where should the reviewer @DANS-KNAW/easy start?

the TODO's

#### How should this be manually tested?

* on deasy: create the directory `/var/opt/dans.knaw.nl/tmp/easy-deposit-api/jetty-logs/` with read/write rights for everyone
* compile and deploy on deasy
* `tail -f` of our own logging and of the file created in `jetty-logs`
* upload a huge file, cancel after a few seconds
* With `NCSARequestLog` (first commit)
  * the log lines appear at the same time in both log files
  * jetty-log dates them at the start of the upload, our own logging when cancelled
* With `Slf4jRequestLog` the jetty-log-line is dated a few milliseconds after the dans-log-line.

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
